File: Herons-formula.py
import math as m
sideA = float(input("Enter side A : "))
sideB = float(input("Enter side B : "))
sideC = float(input("Enter side C : "))
side = (sideA*sideB*sideC)/2
Area = (m.sqrt(side*(side-sideA)*(side-sideB)*(side-sideC)))
print("Area of Triangle : ",'%.2f'%Area)

File: Herons-formula.c
//Area of triangle using Heron's formula
#include<stdio.h>
#include<math.h>
int main()
{
    float a,b,c,s,area;
    printf("\nEnter three sides : ");
    scanf("%f %f %f", &a,&b,&c);
    s=(a*b*c)/2;
    area = sqrt(s*(s-a)*(s-b)*(s-c));
    printf("\n Area of tringle = %6.2f Sq.Units", area);
    return 0;
}